doc/getting-started.rst

```diff
@@ -0,0 +1,596 @@
+=============================
+Getting started with Buildout
+=============================
+
+.. contents::
+
+.. note::
+
+   In the Buildout documentation, we'll use the word
+   *buildout* to refer to:
+
+   - The Buildout software
+
+     We'll capitalize the word when we do this.
+
+   - A particular application of Buildout, which has a buildout
+     configuration.
```

Comment:
I tend to read "application" as "application program", which makes this a bit confusing.  Maybe use more concrete terms, such as "a directory with a `buildout.cfg`"?